packages/blade/src/components/Button/Button/Button.tsx

```diff
@@ -43,6 +43,32 @@ type ButtonCommonProps = {
    * @private
    */
   'aria-describedby'?: string;
+  /**
+   * It is exposed for internal usage with menu.
+   *
+   * @private
+   */
+  'aria-controls'?: string;
+  /**
+   * It is exposed for internal usage with menu.
+   *
+   * @private
+   */
+  'aria-expanded'?: string;
+  /**
+   * It is exposed for internal usage with menu.
+   *
+   * @private
+   */
+  'aria-haspopup'?: 'menu';
+  /**
+   * It is exposed for internal usage with menu.
+   *
+   * @private
+   */
+  role?: 'menuitem';
+  tabIndex?: BaseButtonProps['tabIndex'];
+  id?: BaseButtonProps['id'];
```

Comment:
People will target the button by id. Lets not expose this please 🙈

Reply:
true. The floating UI does pass an id to trigger components. But things seem to work fine without that also so maybe we can skip this 🤔

Reply:
Removed id